docs/quickstart.md

```diff
@@ -0,0 +1,119 @@
+# Quickstart
+
+This page describes how to run, test, and develop PrairieLearn. Ensure that you have already installed PrairieLearn either:
+
+- Natively, by following the [running natively](installingNative.md) guide
+- Via Docker, by following the [running via Docker](installingLocal.md) guide
+
+## Development server
+
+- Run the server in development mode to automatically restart when changes are detected:
+
+  ```sh
+  make dev
+  ```
+
+- In a web-browser go to [http://localhost:3000](http://localhost:3000).
+
+- To stop the server, use ++ctrl+c++.
```

Comment:
Nit: let's be consistent with either using lists or not. I'd err on the side of not.

Reply:
Changed w/o lists in 11571eaf4cb629f088c608b49baefe0e835027bd.